plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDKLaunchController.java

```diff
@@ -227,24 +230,47 @@ private void processTerminated(IServer server,IProcess process, IDebugEventSetLi
 		final ControllableServerBehavior beh = (ControllableServerBehavior)JBossServerBehaviorUtils.getControllableBehavior(server);
 		new Thread() {
 			public void run() {
-				try {
```

Comment:
handleProcessTerminated is launched from a thread, but spawns a new thread itself